docs/taxprofiler_map.md

```diff
@@ -0,0 +1,13 @@
+| Taxprofiler tags                       | Mandatory | HK tags                          | Used by  |
+|----------------------------------------|-----------|----------------------------------|----------|
+| kraken2_combined_report, kraken2       | True      | kraken2, combined-report         | deliver  |
+| kraken2_report, kraken2                | True      | kraken2, metagenomics-report     | deliver  |
+| krona_kraken_plot, krona               | True      | kraken2, krona, visualization    | deliver  |
+| krona_centrifuge_plot, krona           | True      | centrifuge, krona, visualization | deliver  |
+| bracken_combined_report, bracken       | True      | bracken, combined-report         | deliver  |
+| bracken_report, bracken                | True      | bracken, metagenomics-report     | deliver  |
+| centrifuge_combined_report, centrifuge | True      | centrifuge, combined-report      | deliver  |
+| centrifuge_report, centrifuge          | True      | centrifuge, combined-report      | deliver  |
```

Comment:
is this table up to date? 🤔

Reply:
Ahh, I forgot the `storage` tag. Updating!